full/Dockerfile

```diff
@@ -154,6 +154,7 @@ RUN curl -fsSL https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-ins
         mypy autopep8 pep8 pylama pydocstyle bandit notebook \
         twine \
     && sudo rm -rf /tmp/*
+ENV PIP_USER=no
```

Comment:
why unsetting it here whereas its set in [gitpod-layer](https://github.com/gitpod-io/gitpod/blob/24f7b609bf79b247d72d1841282639726bcd5891/components/image-builder/workspace-image-layer/gitpod-layer/alpine/gitpod/.bashrc-append#L11) on top? my first impression is that this won't be effective. 

also, just a reminder, without forcibly enabling `PYTHONUSERBASE` under `/workspace` the backup strategy for python is gone, and prebuilds as well. using a virutal env setup seems to be the best way. it would be great to provide a replacement that works.

Reply:
@AlexTugarev the gitpod-layer was removed https://github.com/gitpod-io/gitpod/pull/4923. I will add the `.bashrc-append` changes here.